docs/fidesops/docs/guides/saas_config.md

```diff
@@ -0,0 +1,370 @@
+## What is a SaaS configuration schema?
+
+A SaaS connector is defined in two parts, the [Dataset](datasets.md) and the SaaS config. The Dataset describes the data that is available from the connector and the SaaS config describes how to connect and retrieve/update the data in the connector. If you contrast this to a [database connector](database_connectors.md), the ways to retrieve/update data conform to a specification (such as SQL) and are consistent. When accessing data from APIs, each application or even different endpoints within the same application can follow different patterns. It was necessary to have a flexible configuration to be able to define the different access/update patterns. Keep in mind that SaaS configs are only applicable to SaaS connector, not database connectors.
+
+In short, you can think of the Dataset as the "what" (what data is available from this API) and the SaaS config as the "how" (how to access and update the data).
+#### An example SaaS config
+
+For this guide, we will use the SaaS config to connect to Mailchimp, this config defines:
+
+- The domain and authentication requirements for an HTTP client to Mailchimp
+- A test request for verifying the connection was set up correctly
+- Endpoints to the following resources within the Mailchimp API:
+    - `GET` and `PUT` for the [members](https://mailchimp.com/developer/marketing/api/list-members/) resource
+    - `GET` for the [conversations](https://mailchimp.com/developer/marketing/api/conversations/) resource
+    - `GET` for the [messages](https://mailchimp.com/developer/marketing/api/conversation-messages/) resource
+
+```yaml
```

Comment:
A piece of text describing this as an example schema for Mailchimp would be useful here